lit_llama/adapter.py

```diff
@@ -94,7 +94,7 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
             ak = ak.view(1, aT, self.n_head, head_size).repeat(B, 1, 1, 1).transpose(1, 2)
             av = av.view(1, aT, self.n_head, head_size).repeat(B, 1, 1, 1).transpose(1, 2)
 
-            ascores = torch.matmul(q, ak.transpose(2, 3)) / math.sqrt(self.n_embd)
+            ascores = torch.matmul(q, ak.transpose(2, 3)) / math.sqrt(head_size)
```

Comment:
This was a small bug. Doesn't break the adaption but the scaling here lead to an inconsistency. I didn't notice this previously in my comparisons because in order for this to be noticable, the gate needs to be > 0 otherwise adaption is effectively disabled.

Reply:
great catch, I actually looked at this last night and it escaped it too